### PR TITLE
PCM-610 Initial MockUp

### PR DESCRIPTION
**Pull request by renujhamtani**
@mattnolting @sshumake @pbathia Please review and let me know the comments.

The topCases.jade ( final output is not looking good ). Please suggest some changes. I am using Leslie's mockup designs[1]

[1] https://mojo.redhat.com/docs/DOC-1019239?et=notification.mention#comment-990568

### CODE REVIEW

File: app/ascension/views/caseQueue.jade [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+    .row
+        .row
+            .col-sm-12(rha-topcases='')
+        section.main.col-md-9
+            .row
+            .row
+                .col-sm-5
+                    a.btn.btn-info.btn-sm(href='#/case/new'){{'Edit'|translate}}
+                    &nbsp;
+                    a.btn.btn-info.btn-sm(translate='', ng-click='', ng-disabled=''){{'Close'|translate}}
```
Comment (pbathia) [inline app/ascension/views/caseQueue.jade]: what is translate inside round bracket mean ? translate should be only at one place for one dom element right ?

Comment (mattnolting) [inline app/ascension/views/caseQueue.jade]: The row nesting is incorrect. Should be .row > (immediate child) .col-sm-12

Comment (mattnolting) [inline app/ascension/views/caseQueue.jade]: Remove this row

**Comment by renujhamtani** [comment]
I have incorporated Matt and Pranjal's comments. I am merging this for now as there are many files to be pushed and this would not hamper any other work.  So pushing this for now. Please review it as and when you get time and i will incorporate those suggestions.

**Pull request merged**